setup.sh

```diff
@@ -33,8 +33,9 @@ else
 cat <<EOF >stacks/${ENV}.tfvars
 remote_state_bucket = "${TERRAFORM_BUCKET}"
 stack_name = "${ENV}"
-dev_environment = "${DEV_ENVIRONMENT}"
+dev_environment = "true"
 prometheus_subdomain = "${ENV}"
+targets_s3_bucket="gds-prometheus-targets-dev"
```

Comment:
Could we maybe store this string as a variable at the top of the script, something like `DEFAULT_DEV_TARGETS_S3_BUCKET`

Reply:
is this just to make it easier to change later on?

Reply:
My thinking was more that it's essentially a configuration value and it's good to have our config and code that does things live separately if possible (whether that be in different files or just separated at the top of the script). It makes it easier to see what all your config are without having to look in random places in the script.